README.md

```diff
@@ -1,6 +1,11 @@
 # mbed OS SDCard Driver (sd-driver) for FAT32 Filesystem Support
 
 
+# Warning
+Starting from mbed-os 5.10 this repository is deprecated. 
+Please refer to mbed-os 5.10 documentation for more detail on how to enable SD support.
```

Comment:
Add links to documentation AND the location of the code, please.

Reply:
done